models/pp_cond_exp_mc_urbanczik.h

```diff
@@ -107,82 +107,140 @@ class pp_cond_exp_mc_urbanczik_parameters
+   of the postsynaptic neuron. The synaptic delay is located behind the synapse
+   and before the dendrite. c) Time trace of the State variables that enter the
+   Urbanczik-Senn rule. Due to the synaptic delay :math:`d`, the presynaptic
+   spike train (top) is combined with a delayed version of the postsynaptic
+   quantities; the dendritic membrane potential (middle) and the somatic spike
+   train (bottom).
+
+
+Parameters
+++++++++++
 
 The following parameters can be set in the status dictionary. Parameters
 for each compartment are collected in a sub-dictionary; these sub-dictionaries
 are called "soma" and "dendritic", respectively. In the list below,
 these parameters are marked with an asterisk.
 
-\verbatim embed:rst
-
-============ ======= ==========================================================
- V_m*        mV      Membrane potential
- E_L*        mV      Leak reversal potential
- C_m*        pF      Capacity of the membrane
- E_ex*       mV      Excitatory reversal potential
- E_in*       mV      Inhibitory reversal potential
- g_L*        nS      Leak conductance
- tau_syn_ex* ms      Rise time of the excitatory synaptic alpha function
- tau_syn_in* ms      Rise time of the inhibitory synaptic alpha function
- I_e*        pA      Constant input current
- g_sp        nS      Coupling between soma and dendrite
- g_ps        nS      Coupling between dendrite and soma
- t_ref       ms      Duration of refractory period
-============ ======= ==========================================================
-\endverbatim
-
-Example:
-See pynest/examples/urbanczik_synapse_example.py.
+============   =====   =====================================================
+ V_m*           mV      Membrane potential
+ E_L*           mV      Leak reversal potential
+ C_m*           pF      Capacity of the membrane
+ E_ex*          mV      Excitatory reversal potential
+ E_in*          mV      Inhibitory reversal potential
+ g_L*           nS      Leak conductance
+ tau_syn_ex*    ms      Rise time of the excitatory synaptic alpha function
+ tau_syn_in*    ms      Rise time of the inhibitory synaptic alpha function
+ I_e*           pA      Constant input current
+ g_sp           nS      Coupling between soma and dendrite
+ g_ps           nS      Coupling between dendrite and soma
+ t_ref          ms      Duration of refractory period
+============   =====   =====================================================
+
+See :doc:`../auto_examples/urbanczik_synapse_example` to learn more.
 
 Remarks:
 
-The neuron model uses standard units instead of the unitless quantities used in [1].
+The neuron model uses standard units instead of the unitless quantities used in
+[1]_.
 
-@note All parameters that occur for both compartments
-are stored as C arrays, with index 0 being soma.
+.. note::
+   All parameters that occur for both compartments are stored as C arrays, with
+   index 0 being soma.
```

Comment:
```suggestion
   index 0 being the soma.
```

Reply:
Considering this is part of user documentation, does the reader need to know the compartment are stored as C arrays?

Reply:
@jessica-mitchell No, this information is only useful for developers, I think.

Reply:
@jstapmanns  in that case I'd suggest to move that note out of this documentation block

Reply:
@jessica-mitchell Yes.